packages/alchemy/src/provider/light-account.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  getDefaultLightAccountFactoryAddress,
+  LightSmartContractAccount,
+} from "@alchemy/aa-accounts";
+import { LightAccountAlchemyProviderConfigSchema } from "../schema.js";
+import type { LightAccountAlchemyProviderConfig } from "../type.js";
+import { AlchemyProvider } from "./base.js";
+
+export const createLightAccountAlchemyProvider = (
```

Comment:
Can we make his a generic type <P extends SmartAccountProvider> and put it to AA-accounts? Thinking if there would be a way for AA-alchemy to not have dep on AA-account...?